### ISSUE TITLE
.clap support?

### ISSUE THREAD

**Issue by RustoMCSpit** [calgoheen/cStop#2]

**Comment by calgoheen** [comment]
Thanks for the suggestion, I have added CLAP support in v1.0.1

**Comment by RustoMCSpit** [comment]
BomBom  cStop.v1.0.1.pkg  PackageInfoPackageInfo  PayloadPayload

What are all of these?

**Comment by calgoheen** [comment]
Yeah, those are the contents of the Mac installer. I will have builds and installers for Windows soon.

**Comment by RustoMCSpit** [comment]
Linux?
